samples/stamp/activeeon/README.md

```diff
@@ -35,39 +33,48 @@ directory to be `samples/stamp/activeeon` and then run:
 docker run -t -v $PWD:/campworkingdir camp camp generate -d /campworkingdir
 ```
 
-You can clean exited container and unlinked images using:
+CAMP then generates new Dockerfiles, properly chained together according to the configurations generated by camp generate. To do so, you can apply the following command:
 
 ```bash
-docker system prune
+docker run -t -v $PWD:/campworkingdir camp camp realize -d /campworkingdir
 ```
 
-## Getting proactive from jenkins
-
-Visit your jenkins site to get the the api-token first.
+You can clean exited container and unlinked images using:
 
 ```bash
-http://<yourserver>/user/<username>/configure
-
-curl --silent --show-error http://<username>:<api-token>@<jenkins-server>/job/<job-name>/doDelete
+docker system prune
 ```
 
-## Getting system tests
+## Test the gerenrated configurations:
 
-* See with Fabien, is it a single test or all?
+Now that the configurations are generated, you need to do some modifications to make it working:
+1- You need to provide your host machine address needed for configuring the proactive server inside docker.
+e.g., on mac you can use the following command to get your ip address: `ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'`
 
-* How to run the tests within camp? see with Franck Chauvel
+2- To get snapshot version of proactive from jenkins, you need to provide the api-token first. Visit the jenkins site `http://<yourserver>/user/<username>/configure` and then generate the token and add it in the get-last-proactive script.
 
+Now you are ready to start the proactive server with two db services: Postgres and MySQL.
 
-## Todo
+In each generated config, please run:
 
-* Entrypoint.sh that find and call proactive server `/root/proactive/bin/proactive-server`
+```bash
+docker-compose up
+```
+## Run system-tests
 
-* After `docker-compose up` on `config_0`, we must have 2 containers running:
-  - 1 mysql-camp
-  - 2 proactive server
+* Now it is time to run system-tests over generated configs.
 
+-Clone the project
+```bash
+git clone https://mboussaa@bitbucket.org/activeeon/scheduling-system-tests.git
```

Comment:
unfortunately, the scheduling-system-tests project is private, and should remain so as it contains sensible credentials. Not sure how we handle this sharing with the public Stamp project

Reply:
I see that your fork is also private. So two questions:
 
 - if you want to use a private repository, why not use the activeeon one rather than your fork

 - if it's private, I think you should mention that it's a private repository and that access should be requested to our team (provide an email address)

Reply:
@fviale Yes true. I have to change it and provide Activeeon repo. Any ways, as for stamp public we need of course to mention that or just say that we run tests locally and in private. I don't think that we really have to share our sys-tests publicly.

Reply:
Ok, then if you could simply add a comment for clarity.

Reply:
@fviale done